### PR TITLE
Sign up page

### PR DESCRIPTION
**Pull request by Samarth-5109**
Screenshot (47)

### CODE REVIEW

File: styles/sign-up.css [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+}
+.sign-up-link {
+  color: #86c6f4;
+  font-weight: bold;
+}
+
+.options {
+  margin-bottom: 10px;
+  font-size: 13px;
+  color: #86c6f4;
```
Comment (sanket4120) [inline styles/sign-up.css:45]: variable names can be created for colors

File: styles/sign-up.css [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+}
+
+.option2 {
+  color: var(--font-primary-color);
+}
+
+.btn {
+  background-color: var(--primary-color);
+  padding: 12px;
+  border-radius: 5px;
```
Comment (sanket4120) [inline styles/sign-up.css:57]: If same border radius is getting repeated at multiple places then create variable to it

File: styles/sign-up.css [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+  display: block;
+}
+
+.option2 {
+  color: var(--font-primary-color);
+}
+
+.btn {
+  background-color: var(--primary-color);
+  padding: 12px;
```
Comment (sanket4120) [inline styles/sign-up.css:56]: Here em can be used for padding as it will scale according to button size

**Review by sanket4120 — APPROVED** [review]
Rest looks good to me

**Review by MonikaJhingan — CHANGES_REQUESTED** [review]
Code looks good to be merged.

**Review by iamharnad — COMMENTED** [review]
great use of classes with naming conventions. usage of colour variables instead of hexcodes. overall looking great. good to go.

**Pull request merged**